src/common/LoadingListItem.js

```diff
@@ -0,0 +1,56 @@
+// @flow
```

Comment:
loading rows will look like this:
![loading](https://user-images.githubusercontent.com/8849922/32870088-e359169e-ca82-11e7-8458-beefe02bde96.png)

Reply:
This is cool. Reminds me of FB :)